core/src/main/java/org/apache/carbondata/core/metadata/schema/table/CarbonTable.java

```diff
@@ -267,20 +267,29 @@ public static CarbonTable buildTable(
 
   private static CarbonFile getFirstIndexFile(CarbonFile tablePath) {
```

Comment:
Change the name of the method, now it is latest index file right

Reply:
done. Please review